src/docfx/build/xref/XrefResolver.cs

```diff
@@ -227,6 +261,26 @@ private void ValidateUIDGlobalUnique()
             }
         }
 
+        private void ValidateExternalXref()
+        {
+            if (!_externalXref.IsValueCreated)
+            {
+                _ = _externalXref.Value;
+            }
```

Comment:
remove that because _externalXref.Value is called below

Reply:
OK.